### PR TITLE
Fix wrong path in desktop file

### PR DESCRIPTION
**Pull request by detiam**
## Description

https://github.com/LizardByte/Sunshine/pull/1817#discussion_r1512688732

## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [ ] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

File: cmake/prep/special_package_configuration.cmake [hunk 1]
```diff
@@ -3,6 +3,8 @@ if (APPLE)
         configure_file(packaging/macos/Portfile Portfile @ONLY)
     endif()
 elseif (UNIX)
+    include(GNUInstallDirs)
```
Comment (ReenigneArcher) [inline cmake/prep/special_package_configuration.cmake:6]:
```suggestion
    include(GNUInstallDirs)  # this needs to be included prior to configuring the desktop files
```

**Review by ReenigneArcher — CHANGES_REQUESTED** [review]
Thank you!

**Comment by tgurr** [comment]
Appears to work fine here with that patch/PR applied to 0.22.0 as well, thanks @detiam.

**Review by ReenigneArcher — COMMENTED** [review]
Should this be replaced in other places as well?

image

**Comment by tgurr** [comment]
> Should this be replaced in other places as well?
> 
> image

https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html#result-variables
Afaik not necessarily, things should work fine for the `install()` command figuring out the paths as detaild in the CMake documentation. It appears just to be a problem for the `configure_file()` where this doesn't work as @detiam found out. (I didn't check other projects how and if they generate their desktop files, maybe there's also a saner way in general.)

**Review by ReenigneArcher — APPROVED** [review]

**Pull request merged**